stable/contour/templates/clusterrole.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.rbac.create -}}
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ template "contour.fullname" . }}
+  labels:
+    app: {{ template "contour.fullname" . }}
+    chart: {{ template "contour.chart" . }}
+    release: {{ .Release.Name }}
```

Comment:
add heritage as well, see: https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md

Reply:
Added heritage to metadata labels.